modules/gltf_blend/config.py

```diff
@@ -0,0 +1,16 @@
+def can_build(env, platform):
+    return not env["disable_3d"]
```

Comment:
This would also need to check that the `gltf` module is enabled.

But isn't it a bit overkill to have a dedicated module for this? It could simply be a file in the `gltf` module, since it depends on the `gltf` importer.

Reply:
If kept as a module, it should also be excluded if `tools` is not enabled.

Reply:
> If kept as a separate module, it should also be excluded if tools is not enabled.

Wouldn't that mean that a headless build used in a CI/CD pipeline wouldn't be able to export projects which import blender files?

Reply:
Headless builds are `tools=yes`, otherwise they can't export at all.

Reply:
Ah, all good then